### PR TITLE
Setup development environment with Docker (NGINX & webpack).

### PR DESCRIPTION
**Pull request by tommyip**
Starting the development environment is just `docker-compose up --build -d`!!
The static contents is hot-reloaded on save, they get bundled by webpack and then stored in a share volume which is shared with NGINX.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "0.1.0",
+    "description": "2D platformer with procedural wave map.",
+    "author": "KryptonChicken",
+    "license": "Apache-2.0",
+    "scripts": {
+        "build": "webpack"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/tommyip/soundzcape.git"
```
Comment (anshuman73) [inline package.json]: Wouldn't it be better to use `https://github.com/KryptonChicken/soundzcape.git` ?

Comment (tommyip) [reply package.json]: Oh yeah, didn't catch that one. I generated the file form `npm init`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    "bugs": {
+        "url": "https://github.com/tommyip/soundzcape/issues"
```
Comment (anshuman73) [inline package.json]: same goes here.

File: package.json (hunk 1 above)
Comment (anshuman73) [reply package.json]: No issues, this is what reviews are for :)

**Comment by tommyip** [comment]
Ready for review!

**Comment by tommyip** [comment]
It would be great if you can test it out locally and tell me if there is anything I can do to improve this.

**Review by anshuman73 — APPROVED** [review]

**Pull request merged**